arm-servicebus/2015-08-01/swagger/servicebus.json

```diff
@@ -2338,49 +2304,45 @@
             "Unknown"
           ],
           "x-ms-enum": {
-            "name": "EntityStatus",
+            "name": "entityStatus",
```

Comment:
Over here it is changed to camelCase and [AvailabilityStatus](https://github.com/Azure/azure-rest-api-specs/pull/472/files#diff-654d402701463525866911e60a6308a1R2222) is changed to PascalCase. Can we get consistency?